### DISCUSSION TITLE
[JointJS]: Svelte question

### DISCUSSION THREAD

**Discussion by huylm2011** [clientIO/joint#2341] [Q&A]
### Introduction

Hi team,

I use svelte to develop my small app. When i see this demo is very impressive https://www.jointjs.com/demos/chatbot . 
But i dont see source code for Svelte. Could you have any source code of this demo for studying?

Thanks Team

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by jamesgeorgewilliams** [comment]
We don't have the Chatbot demo available in Svelte, but you can checkout the following demos written in Svelte.

https://www.jointjs.com/demos/tabs
https://github.com/clientIO/joint-plus-tutorial-svelte/tree/svelte-tabs

https://www.jointjs.com/demos/diagram-index
https://github.com/clientIO/joint-plus-tutorial-svelte/tree/svelte-tree-view

We also have the following Svelte tutorial available too.
https://resources.jointjs.com/tutorial/svelte-ts

**Reply by huylm2011** [reply]
Thanks @jamesgeorgewilliams so much!
